src/main/java/Event.java

```diff
@@ -0,0 +1,37 @@
+class Event extends Task {
+    private final String startDateAndTime;
+    private final String endDateAndTime;
+
+    Event(boolean isDone, String taskDescription, String sdnt, String ednt) {
```

Comment:
Code quality: Name well

Try to use full forms for sdnt and ednt, so that it is clear to the reader and does not cause confusions.